swagger_server/controllers/expression_controller.py

```diff
@@ -35,7 +46,7 @@ def get_expression(expression_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    return Expression.get_by_id(expression_id)
```

Comment:
I've been jsonifying these - I don't think it'll serialize directly w/o something fancy or unpacking em manually.